pandas/tests/indexes/common.py

```diff
@@ -0,0 +1,678 @@
+# -*- coding: utf-8 -*-
+
+from pandas import compat
+from pandas.compat import PY3
+
+import numpy as np
+
+from pandas import (Series, Index, Float64Index, Int64Index, RangeIndex,
+                    MultiIndex, CategoricalIndex, DatetimeIndex,
+                    TimedeltaIndex, PeriodIndex)
+from pandas.util.testing import assertRaisesRegexp
+
+import pandas.util.testing as tm
+
+import pandas as pd
+
+if PY3:
+    unicode = lambda x: x
+
+
+class Base(object):
+    """ base class for index sub-class tests """
+    _holder = None
+    _compat_props = ['shape', 'ndim', 'size', 'itemsize', 'nbytes']
+
+    # @staticmethod
```

Comment:
ahh ok, yeh I saw you added cached setup routines like fixtures in some other test modules